app/styles/partials/nav-bar.scss

```diff
@@ -0,0 +1,4 @@
+.right.menu > .item > .ui.blue.button {
```

Comment:
@anu-007 `right menu` is again a generic semantic UI class https://semantic-ui.com/collections/menu.html
This is not what I meant by wrapping. I meant to say that make it specific to this navbar, right menu may be used some place else too, please change this.